src/components/PlantsV2/components/PlantsReportedPerSpeciesCard.tsx

```diff
@@ -81,13 +82,15 @@ export default function PlantsReportedPerSpeciesCard({
           </Typography>
           <Box>
             <BarChart
+              key={values?.length}
```

Comment:
Could it happen that data changes but not the length?

Reply:
Not the way it is setup today. It's a one time read, data is not refreshed or updated after that.